docs/repositories-configure/using-submodules.md

```diff
@@ -30,11 +28,20 @@ After we enabled submodules for your organization, do the following:
 
 ## Using submodules on new repositories
 
-You must repeat steps 2 to 4 above every time you add new repositories with submodules to Codacy.
+Every time you add new repositories with submodules to Codacy, repeat the steps 2 to 4 [above](#enable).
```

Comment:
I suggest highlighting the sentence fragment with the verb to [frontload the main action](https://codacy.slite.com/app/docs/m4Ynh0i9k8qhqP#ca19f3d3).

```suggestion
Every time you add new repositories with submodules to Codacy, [repeat the steps 2 to 4](#enable) above.
```